src/app/registration/registration.passwordvalidation.ts

```diff
@@ -11,7 +11,7 @@ export class PasswordValidation {
    * (Used for form field validation)
    * @param {string} input1 the first password
    * @param {string} input2 the second password
-   * @returns {AbstractControl} AbstractControl
+   * @returns {any} (AC: AbstractControl) => any
```

Comment:
Ganz im Ernst: Wenn die JSDoc falsch ist, richtet sie nur Schaden an, vor allem, weil die Signatur in TS dank IDE immer bekannt ist. Können wir die bitte weglassen?

Reply:
Geht nicht nur um den Rückgabewert, sondern auch um die Parameter. Dass die falsch benannt & kommentiert sind, ist mir erst jetzt aufgefallen, weil ich mich auf die Dokumentation verlassen habe.

Reply:
Die Parameter passen doch, oder?

Reply:
Ne, input1/2 sind die Namen der Kindelemente des `AbstractControls`, in denen die Passwörter liegen und nicht die Passwörter selbst, wie es in der JSDoc steht. Und als Rückgabetyp ist `any` angegeben, was nicht `(AC: AbstractControl) => any` bzw. `=> null | undefined` ist